electrum_nmc/electrum/gui/qt/configure_name_dialog.py

```diff
@@ -127,7 +127,7 @@ def register_and_broadcast(self, identifier, value, transfer_to):
 
         try:
             # TODO: support non-ASCII encodings
-            name_autoregister(identifier.decode('ascii'), value.decode('ascii'), destination=recipient_address, wallet=self.wallet)
+            name_autoregister(name_to_str(identifier, Encoding.HEX), name_to_str(value, Encoding.HEX), destination=recipient_address, wallet=self.wallet, name_encoding='hex', value_encoding='hex')
```

Comment:
I think you can use `Encoding.HEX` instead of `'hex'` for the `name_encoding` and `value_encoding` arguments, which will yield slightly better type-safety.

Reply:
Nice!